eng/pipelines/aggregate-reports.yml

```diff
@@ -75,3 +75,28 @@ jobs:
   - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@3
     displayName: 'Publish Security Analysis Logs'
     condition: succeededOrFailed()
+
+- ${{if eq(variables['System.TeamProject'], 'internal')}}:
```

Comment:
This pipeline should only ever be in internal so this condition shouldn't be needed.